addressed review comments

eli-w-king opened the pull request:
Added chat history to empty state under setting `chat.emptyState.history.enabled`. Addresses https://github.com/microsoft/vscode/issues/251187

<img width="1728" height="1082" alt="Screenshot 2025-08-19 at 9 27 00 PM" src="https://github.com/user-attachments/assets/15e43aa7-1088-47ff-b053-9a76f3ecb1d1" />

eli-w-king committed:
added chat history empty state under setting

eli-w-king committed:
graunlar time view, mins hrs and days

eli-w-king committed:
chat history link hidden if list is empty

eli-w-king committed:
lowered welcome disclaimers

eli-w-king committed:
fixed lineheight and link color

eli-w-king committed:
disclaimer spacing

eli-w-king committed:
disclaimer spacing

eli-w-king committed:
Merge branch 'main' into eli/chat-history-empty-state

eli-w-king committed:
Merge branch 'main' into eli/chat-history-empty-state

Agingvollkkaa99 reviewed (commented):
Leave a review

bpasero commented:
Are we still working on the UX? Somehow it is not immediately clear to me that the top thing is about previous chats.

justschen commented:
> Are we still working on the UX? Somehow it is not immediately clear to me that the top thing is about previous chats.

@bpasero i think this was what landed, but can maybe discuss some more in ux sync if we think it's not clear enough

eli-w-king committed:
Hide chat history and plus buttons when empty state history is enabled and chat is empty

- Only hide buttons when chat.emptyState.history.enabled is true AND chat has no items
- Buttons always show in active chat sessions regardless of setting
- Added inEmptyStateWithHistoryEnabled context key to track combined condition
- Updated menu conditions for both new chat and history actions

eli-w-king committed:
Fix whitespace indentation issues in chatWidget.ts

eli-w-king commented on src/vs/workbench/contrib/chat/browser/chatWidget.ts:
```diff
@@ -801,6 +804,12 @@ export class ChatWidget extends Disposable implements IChatWidget {
 				welcomeContent.tips = tips;
 			}
+			// Optional: recent chat history above welcome content when enabled
+			const showHistory = this.configurationService.getValue<boolean>(ChatConfiguration.EmptyStateHistoryEnabled) === true;
```
removed === true

eli-w-king committed:
fixed  telemtry text, setting disabled by default

eli-w-king committed:
added setting render listener

eli-w-king committed:
resolved constants.ts conflict

eli-w-king committed:
resolved showThinking error

eli-w-king committed:
Merge remote-tracking branch 'origin/main' into eli/chat-history-empty-state
